PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -4329,7 +4329,9 @@ void Finder::setFinderStyle()
 		const Style * pStyle = pStyler->findByID(SCE_SEARCHRESULT_CURRENT_LINE);
 		if (pStyle)
 		{
-			_scintView.execute(SCI_SETCARETLINEBACK, pStyle->_bgColor);
+			_scintView.execute(SCI_SETELEMENTCOLOUR, SC_ELEMENT_CARET_LINE_BACK, pStyle->_bgColor);
+			_scintView.execute(SCI_SETCARETLINEFRAME, 0);
+			_scintView.execute(SCI_SETCARETLINEVISIBLEALWAYS, true);

```

Comment:
I don't get it, in https://github.com/notepad-plus-plus/notepad-plus-plus/commit/d3e77c0e6f454374ad5e803da4c35fde9ce7ef45, only _mainEditView & _subEditView are applied by the setting of current line indicator. How come _scintView is affected?